tests/test_cms_config.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+from django.core.exceptions import ImproperlyConfigured
+
+from cms import app_registration
+from cms.models import Page
+from cms.utils.setup import setup_cms_apps
+
+from djangocms_references.cms_config import ReferencesCMSExtension
+from djangocms_references.test_utils.app_1.models import TestModel1, TestModel2
+from djangocms_references.test_utils.app_2.models import TestModel3, TestModel4
+from djangocms_references.utils import supported_models
+
+from .utils import TestCase
+
+
+class AppRegistrationTestCase(TestCase):
+    def test_missing_cms_config(self):
+        extensions = ReferencesCMSExtension()
+        cms_config = Mock(
+            djangocms_referencess_enabled=True, app_config=Mock(label="blah_cms_config")
+        )
+
+        with self.assertRaises(ImproperlyConfigured):
+            extensions.configure_app(cms_config)
+
+    def test_invalid_cms_config_parameter(self):
```

Comment:
Ditto - a docstring or better test name would be helpful. I need to look into the code to know that `reference_models` should be a dict, can't deduce this from the test.